src/main/java/com/metamx/emitter/core/HttpEmitterConfig.java

```diff
@@ -34,19 +37,19 @@
 
   @Min(1)
   @JsonProperty
-  private long flushMillis = 60 * 1000;
+  private long flushMillis = DEFAULT_FLUSH_MILLIS;
 
   @Min(0)
   @JsonProperty
-  private int flushCount = 500;
+  private int flushCount = DEFAULT_FLUSH_COUNTS;
 
   @Min(0)
   @JsonProperty
   private long flushTimeOut = DEFAULT_FLUSH_TIME_OUT;
 
   @NotNull
   @JsonProperty
-  private String recipientBaseUrl = null;
+  private String recipientBaseUrl = DEFAULT_REIPIENT_BASE_URL;
```

Comment:
it is marked as NotNull, I don't think we need a default = `null` for it. 
Let's make Builder constructor requiring at least one parameter `recipientBaseUrl `
It might also make sense to add `flushMillis` and `flushCount` as a required parameters as they were previously set for all (but default) constructors from outside. It is up to you.